tools/cpp_build/libtorch/CMakeLists.txt

```diff
@@ -10,6 +10,11 @@ if (VERBOSE)
   message(STATUS "ATEN_BUILD_PATH is ${ATEN_BUILD_PATH}")
 endif()
 
+set(CMAKE_CXX_FLAGS "--std=c++11 -Wall -Wextra -Wno-unused-parameter -Wno-unused-variable ${CMAKE_CXX_FLAGS}")
```

Comment:
you don't have to change this, but cmake YI; list(APPEND ...) is faster than set().

Reply:
I don't think this works -- `CMAKE_CXX_FLAGS` needs to be a (string separated) string, not a list

Reply:
You should prefer TARGET_COMPILE_OPTIONS I think.